package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.1.0",
   "author": "Zain Fathoni <zain.fathoni@gmail.com>",
   "dependencies": {
-    "@emotion/core": "^10.0.10",
+    "@emotion/core": "10.0.10",
```

Comment:
@ri7nz I used to check the availability of the caret (`^`) operator manually and I missed it on your last PR.
Thanks to @renovate-bot, we don't have to do it manually anymore! 🎉

Reply:
I think to Renovate fully help for little & lot case like this. @zainfathoni